modeltranslation/manager.py

```diff
@@ -447,7 +448,25 @@ def values_list(self, *fields, flat=False, named=False):
         if not fields:
             # Emulate original queryset behaviour: get all fields that are not translation fields
             fields = self._get_original_fields()
-        clone = self._values(*fields, prepare=True, selects_all=selects_all)
+
```

Comment:
Fyi: this is based on https://github.com/django/django/blob/2fd755b361d3da2cd0440fc9839feb2bb69b027b/django/db/models/query.py#L1318-L1337